tests/python/frontend/tflite/test_forward.py

```diff
@@ -2480,6 +2481,16 @@ def __test_elemwise(in_data):
                 inq0_min, inq0_max = (out_min, out_max)
                 inq1_min, inq1_max = (out_min, out_max)
 
+            if exclude_zero_point:
+                if inq1_max == inq1_min:
+                    raise ZeroDivisionError("Input range is 0.")
+
+                # only compute for rhs.
+                quant_scale = 255 / (inq1_max - inq1_min)
```

Comment:
Shouldn't there be a dependency on the data type for the value 255? Since the data type can be int8, int16.